openlibrary/plugins/openlibrary/js/utils.js

```diff
@@ -1,4 +1,4 @@
-;(function($) {
+(function($) {
```

Comment:
(Above)

Reply:
okay, I have added `;` back and updated the PR.